### DISCUSSION TITLE
Unable to Generate PDF Due to Web-to-Print Bundle Version Requirement

### DISCUSSION THREAD

**Discussion by KasthuriAnandhan** [pimcore/pimcore#17509] [Q&A / Support]
Hi All,

I successfully generated a PDF using the PrintPage and PrintContainer on my local machine. However, when attempting to do the same on our Pimcore development site, I encountered an issue while installing the web-to-print bundle. The bundle requires Pimcore version 11 or above, but our current Pimcore version is v10.5.21.

Is there an alternative method or workaround to achieve PDF generation using the web-to-print functionality with our current Pimcore version? Any guidance or suggestions would be greatly appreciated.

Thanks,
Kasthuri Anandhan

**Comment by kingjia90** [comment]
Hi Kasthuri,
in Pimcore 10, the Web to print is shipped out of the box therefore you don't need to install any bundle.
Just go to Settings and enable it, please check this out (especially the first screenshot)
https://pimcore.com/docs/pimcore/10.6/Development_Documentation/Documents/Document_Types/Print_Documents.html

**Comment by KasthuriAnandhan** [comment]
Hi @kingjia90 ,

I have enabled what you said, but then also I can't able to see the print pages under documents, see my below screenshots.

![Screenshot 2024-08-21 103941](https://github.com/user-attachments/assets/ab26182a-4fbe-4d24-b886-a103b0b17cbc)
![Screenshot (339)](https://github.com/user-attachments/assets/85a8a75c-52e3-46e4-b104-e25b244351a4)

**Reply by kingjia90** [reply]
Please try to reset the cache

**Comment by KasthuriAnandhan** [comment]
Hi @kingjia90,

Cleared the cache also, but now also the printpage not available.

**Comment by KasthuriAnandhan** [comment]
Hi @kingjia90 

I have done all the things as you said, but then also the print pages are not enabled, this is the most priority one in our project could you please guide me to complete this.

**Reply by kingjia90** [reply]
Unfortunately, i can't provide any further help (10.5 it's not a supported version anymore) other than maybe recommending to upgrade to latests 10.x version to rule out that the specific pimcore version you are using has a problem.
The instructions should be straightforward, maybe please double check if custom view and perspective.

**Comment by KasthuriAnandhan** [comment]
Hi @kingjia90 
Is the Custom Views and Perspectives are mandatory to enable print pages, because I didn't created any custom views and perspectives.
